app/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,5 @@
     <string name="password">Password</string>
     <string name="confPass">Confirm Password</string>
     <string name="register">Submit Registration</string>
+    <item name="button_login" type="id">button_login</item>
```

Comment:
+    <item name="button_login" type="id">button_login</item>
This is the wrong format for string values in strings.xml. Look at how the other strings are formatted. You also give the string the same value as its key. THe value should be what you actually want displayed.